sdk/textanalytics/Azure.AI.TextAnalytics/src/NamedEntity.cs

```diff
@@ -18,41 +22,41 @@ internal NamedEntity(string text, string type, string subType, int offset, int l
         }
 
         /// <summary>
-        /// Entity text as appears in the request.
+        /// Gets the entity text as it appears in the input document.
         /// </summary>
         public string Text { get; }
 
         /// <summary>
-        /// Gets entity type from Named Entity Recognition model.
-        /// Entity type, such as Person/Location/Org/SSN etc.
+        /// Gets the entity type inferred by the text analytics service's
+        /// named entity recognition model.  The list of available types is
+        /// described at
+        /// https://docs.microsoft.com/en-us/azure/cognitive-services/Text-Analytics/named-entity-types.
         /// </summary>
         public string Type { get; }
 
         /// <summary>
-        /// Gets entity sub type from Named Entity Recognition model.
-        /// Entity sub type, such as Age/Year/TimeRange etc.
+        /// Gets the sub type of the entity inferred by the text analytics service's
```

Comment:
We should mention this is an optional field. For some types we may not have a sub-type.